CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Malformatted table cell for analysis date on caseS page (#5438)
 - Remove "Add to ClinVar submission" button for pinned MEI variants as submission is not supported at the moment (#5442)
 - Clinical variant files could once again be read in arbitrary order on load (#5452)
+- Fix test_sanger_validation test to be run in the app context (#5412)
```

Comment:
```suggestion
- Fix test_sanger_validation test to be run with a mock app instantiated (#5453)
```

Reply:
It's not really that it is in context, is it. But with adding the fixture, you ensure that the mock app init has been done, and that will as it happens call the app setup on the named database, and creating the collection variables. Which is the same as is used in the test.

Reply:
Yes I understand. Good suggestion! 👍🏻